app/modules/raffle/endpoints_raffle.py

```diff
@@ -189,9 +189,9 @@ async def get_raffle_stats(
     tickets = await cruds_raffle.get_tickets_by_raffleid(db=db, raffle_id=raffle_id)
 
     tickets_sold = len(tickets)
-    amount_raised = sum(
+    amount_raised = int(sum(
         [ticket.pack_ticket.price / ticket.pack_ticket.pack_size for ticket in tickets]
-    )
+    ))
```

Comment:
The goal of this PR is to avoid imprecision errors. Maybe there is a way to avoid dividing values before converting to int